### ISSUE TITLE
Fix WCAG 1.4 Distinguishable issues

### ISSUE THREAD

**Issue by juliatufts** [scottrigby/basekamp-next#6]
Fix the following issues related to WCAG 1.4: Distinguishable, see also: [full accessibility report](https://github.com/basekamp/basekamp-next/wiki/Accessibility-Evaluation-Report-2026).

## All Pages

Issue 1: Links are only distinguished by color.

Suggested Fix 1: Display link with underline.

Issue 2: The following elements fail to meet color contrast:

* Nav bar unselected items (white on orange)
* links within page content (orange on white)
* CC BY-NC-SA 4.0 footer link (orange on grey)
* Basekamp logo in header

Suggested Fix 2: Adjust colors to meet 3:1 contrast.
